### PR TITLE
[ Feat ] Support Global Install

### LINKED ISSUE #44: [ Feat ] Add support for global composer install

### PR DESCRIPTION
**Pull request by ProjektGopher**
Resolves #44 

TODO
- [x] Move `skip-once` to `.git/hooks` to accommodate global install (requires test update as well)
- [x] Update `README.md` with global install instructions (and remove warning)

### CODE REVIEW

File: app/Platform.php [hunk 1]
```diff
@@ -24,6 +24,16 @@ public static function normalizePath(string $path): string
...
     }
 
+    public static function getGlobalComposerHome(): string
+    {
+        return rtrim(shell_exec('composer -n config --global home'), "\n");
+    }
+
+    public static function getGlobalComposerBinDir(): string
+    {
+        return rtrim(shell_exec('composer -n global config bin-dir --absolute --quiet'), "\n");
```
Comment (faissaloux) [inline app/Platform.php:34]:
```suggestion
        return rtrim(shell_exec('composer -n config --global bin-dir --absolute --quiet'), "\n");
```

Comment (faissaloux) [inline app/Platform.php:34]:
```suggestion
        return rtrim(shell_exec('composer -n config --global bin-dir --absolute --quiet'), "\n");
```

**Review by faissaloux — CHANGES_REQUESTED** [review]
Change to make it consistent with the first command.

File: app/Platform.php (hunk 1 above)
Comment (ProjektGopher) [reply app/Platform.php:34]: the `--global` flag actually behaves differently (at least on my system, lol). We might need to change the other spot I use a global cli command instead

Comment (ProjektGopher) [reply app/Platform.php:34]: This is a weird one. The `composer global home` command works as expected using both methods, but trying the `bin-dir` command in tinkerwell on another project returns unexpected results:

```php
rtrim(shell_exec('composer -n config --global bin-dir --absolute --quiet'), "\n"); // "/Users/gopher/Code/EXACTsports/camps/vendor/bin"
rtrim(shell_exec('composer -n global config bin-dir --absolute --quiet'), "\n"); // "/Users/gopher/.composer/vendor/bin"
```

Comment (faissaloux) [reply app/Platform.php:34]: Yeah yeah I just realised that, I have updated it at least keep the order `config - global` same as the one before.

Comment (ProjektGopher) [reply app/Platform.php:34]: playing around in tinkerwell again it looks like it's import that the `global` keyword comes first. I might just change the other command to match if it works the same

Comment (faissaloux) [reply app/Platform.php:34]: Yeah `global` should come first in both cases

File: app/Platform.php [hunk 2]
```diff
@@ -24,6 +24,16 @@ public static function normalizePath(string $path): string
         return $path;
     }
 
+    public static function getGlobalComposerHome(): string
+    {
+        return rtrim(shell_exec('composer -n global config home'), "\n");
```
Comment (faissaloux) [inline app/Platform.php:29]:
```suggestion
        return rtrim(shell_exec('composer -n global config home --quiet'), "\n");
```

**Review by faissaloux — CHANGES_REQUESTED** [review]

File: app/Platform.php (hunk 2 above)
Comment (ProjektGopher) [reply app/Platform.php:29]: I can't see a difference in output when using the `quiet` option. Does it actually even do anything?

Comment (faissaloux) [reply app/Platform.php:29]: Yeah it does actually, on my case it's showing some deprecation warnings, so with `--quiet` it's just showing the wanted result.

Comment (ProjektGopher) [reply app/Platform.php:29]: oh ok, cool. I'll add that in

**Pull request merged**